docker-compose.yml

```diff
@@ -98,8 +101,9 @@ services:
     environment:
       - REDIS_URL=redis://redis:6379/1
       - SQLALCHEMY_DATABASE_URI=postgres://postgres:chummy@db:5432/test_notification_api
+    entrypoint: local/scripts/notify-worker-entrypoint.sh
     command: >
-      bash -c "./scripts/run_tests.sh"
+      bash -c "/app/scripts/run_celery.sh"
```

Comment:
Why are you running this command in the image `notification-api-tests`?

Reply:
Good catch, my mistake. A copy/paste gone wrong probably.